docs/src/main/asciidoc/smallrye-graphql-client.adoc

```diff
@@ -230,10 +230,18 @@ that the target of the client is the application that is being tested (typically
 This is useful if your application contains a GraphQL server-side API as well as a GraphQL client that is used for
 testing the API.
 
+Be aware that the `endpoint` parameter in `@GraphQLClientApi` will win over the configuration in the `application.properties` file.
```

Comment:
Are you saying that URL in the annotation has precedence over what is set in the properties? That would be a bug IMO.

Reply:
According to my testing: yes

I will revert this line in this PR and open an issue for discussion.